src/routes/auth.js

```diff
@@ -1,6 +1,18 @@
 const express = require('express');
 const authController = require('../controllers/auth');
+const {
+  userLoginValidationRules,
+  errorHandlingForValidation,
+} = require('../middleware/validation');
 
 const router = express.Router();
 
+router.post(
+  '/login',
+  userLoginValidationRules,
+  errorHandlingForValidation,
+  authController.loginUsers
+);
+router.get('/profile', authController.getProfile);
```

Comment:
I think it makes more sense for the profile router to be in the `/api/use/` endpoint.

Reply:
I was actually confused about this, but in some of the websites, I saw that it was under auth. But lets open this to discussion